qemu/tests/numa_prealloc_threads.py

```diff
@@ -7,6 +7,18 @@
 from avocado.utils import process
 
 
+def check_affinity(affinity, cmd_taskset, test):
+    """
+    :param affinity: the cpu affinity
+    :param cmd_taskset: the taskset command
+    :param test: QEMU test object
+    """
+    output = process.getoutput(cmd_taskset)
+    if not re.search(affinity, output):
+        test.fail("The affinities %s and %s do not match!"
+                  % (affinity, str(output)))
```

Comment:
`process.getoutput` returns string, why convert output to string again here?

Reply:
You're right, code updated